### ISSUE TITLE
installation failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#147]
```
What steps will reproduce the problem?
1. double click
2. click continue at the certificate window popup
3. continue install till the end when it says "installation failed"

What is the expected output? What do you see instead?

installation failed


What version of the product are you using? On what operating system?

1.6.4 osx 10.8 (updated from 10.6.8)


Please provide any additional information below.

it wont install, i also tried previous version but soundflowerbed crashes every 
time i select an audio source. and after trying installing 1.6.4 it wont let me 
install 1.6.3. ( havent restart the machine yet though).

thank you
```

Original issue reported on code.google.com by `ji...@innovativelanguage.com` on 27 Jul 2012 at 1:20

**Comment by GoogleCodeExporter** [comment]
```
should be fixed in 1.6.5
```

Original comment by `t...@electrotap.com` on 31 Jul 2012 at 6:47
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
not fixed
```

Original comment by `dyan...@gmail.com` on 4 Nov 2012 at 11:53
